tests/FakeItEasy.Specs/StringExtensions.cs

```diff
@@ -53,6 +53,20 @@ public static IStepBuilder See<T>(this string text, Func<T, Action<Type, IFakeOp
             return text.x(() => { });
         }
 
+        /// <summary>
+        /// Creates a step builder that can be used to refer to a type's method.
+        /// Useful when we want to establish a condition with actually performing an action.

```

Comment:
"without actually", I think

Reply:
Actually, I copied it from the above method, which I believe you wrote... so it's your fault 😛
I'll fix it, and the others as well.

Reply:
That does look like the sort of error I would make. Your story checks out. Thanks.